huxley/www/js/components/AdvisorFeedbackView.js

```diff
@@ -172,21 +172,16 @@ var AdvisorFeedbackView = React.createClass({
     assignment to delegates via this object.
   */
 
-  prepareFeedback: function(delegates, assignments) {
-    if (delegates && assignments) {
+  prepareFeedback: function(delegates) {
+    if (delegates) {
       var feedback = {};
       for (var i = 0; i < delegates.length; i++) {
```

Comment:
```
for (var delegate of delegates) {
...
}
```
and updates your individual delegate references accordingly in the body of the for loop